changelog/20595.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Add possibility to decode a generated encoded root token via the rest API
```

Comment:
```suggestion
core: Add possibility to decode a generated encoded root token via the rest API
```

Reply:
It's great to work with such a very fast answering contributor!
done

Reply:
It perfectly matched my flow!